test/unit/util.test.ts

```diff
@@ -60,3 +60,10 @@ test('Encoding ArticleId for Zim HTML Url', async(t) => {
         t.equal(encoded, encodeArticleIdForZimHtmlUrl(unencoded), `encodeArticleIdForZimHtmlUrl() encoding`);
     }
 });
+
+test('wikitext comparison', async(t) => {
```

Comment:
Good, but could you please abtract this with a function taking arguments `wikicode`, `html` and `comment`?

Reply:
I have extracted the function, please let me know if that is how it intends to be?

Reply:
Yes, in principe but:
* Don't use temporary variable, just put the strings in the fonction calls
* Reduce the size of your wikitext/html you test, keep only what is necessary to be tested. It does not have to be a senteence which is online available as well
* rename the fucntion to `testHtmlRewritingE2e()`